rds/instance-base-database.ts

```diff
@@ -31,6 +31,13 @@ export interface InstanceBaseDatabaseProps {
   // if set will override the allocated storage for the db - otherwsie
   // we will have this set to smallest database size allowed (20 Gib)
   overrideAllocatedStorage?: number;
+
+  // Allow monitoring features such as postgres logs exported to cloudwatch and performance insights.
+  enableMonitoring?: {
+    cloudwatchLogsExports: string[],
+    enablePerformanceInsights: true,
+    monitoringInterval: Duration,
+  };
```

Comment:
If this goes directly into DatabaseInstance props - is there not an existing AWS defined type for this?

Reply:
I don't think so, and the configuration is different between the serverless and instance database.